adage-front/src/app/components/OffersSearch/OffersSearch.tsx

```diff
@@ -15,6 +15,9 @@ import { OfferFilters } from './OfferFilters/OfferFilters'
 import { Offers } from './Offers/Offers'
 import Pagination from './Offers/Pagination'
 import { SearchBox } from './SearchBox'
+import {useState} from "react";
+import {facets} from "../../types/facets";
+import {initialize} from "jest-circus/build/legacy-code-todo-rewrite/jestAdapterInit";
```

Comment:
c'est quoi ca ?

Reply:
😱 je ne sais pas, il est inutilisé chez moi, il y a dû avoir un auto-import 😔

Reply:
Wath the... comme il est arrivé là ce truc 🥺